src/Formatter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+declare(strict_types = 1);
+
+namespace LanguageServer;
+
+use LanguageServer\Protocol\{TextEdit, Range, Position, ErrorCode};
+use AdvancedJsonRpc\ResponseError;
+
+class Formatter
+{
+
+    /**
+     *
+     * @param string $content            
+     * @param string $uri            
+     *
+     * @return \LanguageServer\Protocol\TextEdit[]
+     * @throws \AdvancedJsonRpc\ResponseError
+     */
+    public function format(string $content, string $uri)
+    {
+        $path = uriToPath($uri);
+        $cs = new \PHP_CodeSniffer();
+        $cs->initStandard($this->findConfiguration($path));
+        $file = $cs->processFile(null, $content);
+        $fixed = $file->fixer->fixFile();
+        if (!$fixed && $file->getErrorCount() > 0) {
+            throw new ResponseError('Unable to format file', ErrorCode::INTERNAL_ERROR);
```

Comment:
We don't need to throw a `ResponseError`, any Exception will be turned into an `ErrorResponse` with `ErrorCode::INTERNAL_ERROR` by the dispatcher automatically. No need to couple it to JSON RPC here